pkg/exchange/coinbase/convert.go

```diff
@@ -20,6 +21,31 @@ func toGlobalSide(cbSide api.SideType) types.SideType {
 	return types.SideTypeNone
 }
 
+func toGlobalOrderStatus(order *api.Order) types.OrderStatus {
+	switch order.Status {
+	case api.OrderStatusRejected:
+		return types.OrderStatusRejected
+	case api.OrderStatusReceived:
+		return types.OrderStatusReceived
+	case api.OrderStatusOpen, api.OrderStatusPending:
+		return types.OrderStatusNew
+	case api.OrderStatusDone:
+		switch order.DoneReason {
+		case "filled":
+			if order.FilledSize.Eq(order.Size) {
+				return types.OrderStatusFilled
+			} else {
+				return types.OrderStatusPartiallyFilled
```

Comment:
return types.OrderStatusCanceled for this line?

Reply:
https://docs.cdp.coinbase.com/exchange/docs/websocket-channels#done

It should be `filled`.
No partially filled for done message.
Will fix it.